tests/unit/initializers/inject-notifications-test.js

```diff
@@ -1,4 +1,3 @@
-import Ember from 'ember';
```

Comment:
I assume the intention was good with these tests.

Reply:
I'm not sure what you mean. The Ember import is deprecated and should not be used anymore. These tests were generated a bunch of versions back and had to be manually updated to comply with the latest standards.